assets/css/amp-editor-story-blocks.css

```diff
@@ -438,6 +439,32 @@ div[data-type="amp/amp-story-page"] .editor-inner-blocks .editor-block-list__lay
 	opacity: 1;
 }
 
+/*
+ * 9. General block editor components
+ */
+
+.editor-amp-story-page-background {
+	display: block;
+}
+
+.editor-amp-story-page-background + .is-destructive {
+	margin-top: 1em;
+}
+
+.editor-amp-story-page-wrap {
+	position: relative;
+}
+
+.editor-amp-story-page-video {
+	position: absolute;
+	top: 0;
+	left: 0;
+	right: 0;
+	bottom: 0;
+	min-width: 100%;
+	min-height: 100%;
+}
+
```

Comment:
It should be possible to make the video fill the content, maybe with assigning `object-fit:cover` to the `video` element and then `overflow:hidden` to the relevant parent. I think something similar was previously done (not by me) for the `AMP Background Video Layer` so it showed nicely.